manim/mobject/coordinate_systems.py

```diff
@@ -256,28 +265,86 @@ def create_axis(self, range_terms, axis_config, length):
             axis.shift(-axis.number_to_point(0))
         return axis
 
-    def coords_to_point(self, *coords):
+    def coords_to_point(self, *coords: Sequence[float]) -> np.ndarray:
+        """Returns back a point with respect to the origin from the given coordinates.
+
+        Returns
+        -------
+        np.ndarray
+            A point formed by the coordinates
```

Comment:
It's not a point formed by the coordinates (In that case, it'd mean that this function does nothing). It's a point that results from a change of basis from the coordinate system defined by the `Axes` to that of `manim`'s default coordinate system